src/server/pbsd_init.c

```diff
@@ -1709,11 +1710,15 @@ pbsd_init_job(job *pjob, int type)
 			(pjob->ji_qs.ji_un.ji_exect.ji_momaddr != 0)) {
 
 			if (pjob->ji_wattr[(int)JOB_ATR_exec_host].at_flags & ATR_VFLAG_SET) {
-				pjob->ji_qs.ji_un.ji_exect.ji_momaddr =
+				new_momaddr =
```

Comment:
@vchlum thank you for change, change look good, just minor comment: it seems like we are not using new_momaddr and new_momport outside this if condition, so can we reduce its scope to this if condition only? also, can we add some debug log saying like failed to find new mom addr at DEBUG3 or up-level? so it can help in debugging in the future.

Reply:
Thank you for comments @hirenvadalia. Done.